lib/algrep.gi

```diff
@@ -271,16 +271,10 @@ true,
         0,
     function( A, B, opl, opr, gens, str )
 
-      local F,V;
-
       if str<>"basis" then
-         Error( "Usage: RightAlgebraModule( <A>, <op>, <gens>, <str>) where the last argument is the string \"basis\"" );
+         Error( "Usage: BiAlgebraModuleByGenerators( <A>, <B>, <opl>, <opr>, <gens>, <str>) where the last argument is the string \"basis\"" );
       fi;
-
-      F:= LeftActingDomain( A );
-      V:= VectorSpace( F, str );
```

Comment:
I am not sure this is the right way to do things, just ignoring `str`. Rather, I think the correct fix would be to keep the code in this method as it is but replace `V:= VectorSpace( F, str );` by `V:= VectorSpace( F, gens, str );`

Reply:
Aha, now that makes more sense